### ISSUE TITLE
jaws can pry bolted doors

### ISSUE THREAD

**Issue by deltanedas** [space-wizards/space-station-14#22460]
## Description
it kinda makes sense like for saving from car wreckages but its shitty

**Comment by LordEclipse** [comment]
Intended, but also otherwise bolted unpowered doors are impossible to get through unless you outright destroy them.

**Comment by deltanedas** [comment]
not impossible, if you have access you can link something to unbolt it then do whatever

also this means its impossible to keep a door sealed as a traitor e.g. on evac, since someone can just jaw it open

**Comment by metalgearsloth** [comment]
> Intended, but also otherwise bolted unpowered doors are impossible to get through unless you outright destroy them.

Invalidates hacking as a pretty central mechanic for traitors.

**Comment by Ilya246** [comment]
if you bothered to buy jaws and find space for them in your inventory it should be fine that you're able to pry open bolted doors, it's not particularly fast either, hacking may actually be faster

**Comment by JoeHammad1844** [comment]
jaws are normal size last i checked

**Comment by Kadeo64** [comment]
jaws also go into toolbelts + good scientists will always have JOL on standby

**Comment by Kadeo64** [comment]
> jaws are normal size last i checked

2x2 and the same size as a crowbar I think

**Comment by Kadeo64** [comment]
> also this means its impossible to keep a door sealed as a traitor e.g. on evac, since someone can just jaw it open
the only people with jaws of life are
-Chief Engineer
-Research Director (if science team has it)
-Science team 
On evac, CE is probably not even on it in the first place, RD is hiding in bridge, and the science team is too lazy to use their Jaws of Life. Also just weld the door

**Comment by deltanedas** [comment]
anyone can unweld near instantly

**Comment by JoeHammad1844** [comment]
> anyone can unweld near instantly

owie my eyes ow!

**Comment by deltanedas** [comment]
vs get to hunt a valid mmmmmmmmm

**Comment by JoeHammad1844** [comment]
LRP players trying not to gib the latejoin for cybersun pen vs the same LRP player when liltenhead gibs somebody infront of them
![image](https://github.com/space-wizards/space-station-14/assets/130668733/5a58dcc0-f16a-49b7-9afe-c59e7adf4f6b)
![image](https://github.com/space-wizards/space-station-14/assets/130668733/fc92c286-2e27-48b2-a56b-667ea091a445)

**Comment by deltanedas** [comment]
anyway i made this issue because i was being naughty and did some trolling
1. fishops and gib ce
2. declared valid
3. friend emags into evac bridge so i tell him to get item i left for him (he did not pro)
4. take apart window
5. unbolt close and rebolt door, then turn off apc
6. weld it
7. validhunters immediately shout to unweld and then its jawed open instantly
8. get shot at a bit then finished off by salv validhunting in space with axe real

**Comment by Ilya246** [comment]
i think we should make syndicate jaws 2x4 and make exclusively syndicate jaws pry open bolted doors

**Comment by JoeHammad1844** [comment]
> i think we should make syndicate jaws 2x4 and make exclusively syndicate jaws pry open bolted doors

but regular jaws 2x2

**Comment by JoeHammad1844** [comment]
and it would suck to store in belts

**Comment by deltanedas** [comment]
make regular jaws 2x4 and keep valid jaws 2x2

and yes that seems like a nice thing for syndies
